test/e2e/nutanix_client.go

```diff
@@ -50,9 +50,9 @@ type nutanixCredentials struct {
 
 func getNutanixCredentialsFromEnvironment() nutanixCredentials {
 	nutanixUsername := os.Getenv(nutanixUserKey)
-	Expect(nutanixUsername).ToNot(BeNil())
+	Expect(nutanixUsername).ToNot(BeEmpty(),"expected environment variable %s to be set", nutanixUserKey)
 	nutanixPassword := os.Getenv(nutanixPasswordKey)
-	Expect(nutanixPassword).ToNot(BeNil())
+	Expect(nutanixPassword).ToNot(BeEmpty(),"expected environment variable %s to be set", nutanixPassword)
```

Comment:
parameter should be nutanixPasswordKey and not nutanixPassword at the end of this line

Reply:
Good catch. Modified